src/platform/test/platform.spec.ts

```diff
@@ -64,6 +64,30 @@ describe('Platform', () => {
 
   });
 
+  describe('orientation', () => {
+    it('Should return true if orientation is landscape', () => {
+      expect(plt.isLandscape()).toEqual(true);
+    });
+
+    it('Should return false if orientation is not portrait', () => {
+      expect(plt.isPortrait()).toEqual(false);
+    });
+
+    it('Should return a number that is equal to window.innerWidth', () => {
+      let type = typeof plt.width();
+
+      expect(type).toEqual('number');
+      expect(plt.width()).toEqual(window.innerWidth);
```

Comment:
Just added some tests for this in this commit https://github.com/driftyco/ionic/pull/10084/commits/d24435f212677cbbc5b9d576dedabc3530f1ff5e